docs/tutorials/01_basics/03_math.md

```diff
@@ -0,0 +1,130 @@
+
+* An integer literal is written as a series of one ore more base-10 digits (`0-9`) without leading zeros. The default type is `Int32`.
+* A floating point literal is written as a series of two or more base-10 digits (`0-9`) with a point (`.`) somewhere in the middle,
+  indicating the decimal point. The default type is `Float64`.
+
+All numeric types allow underscores at any place in the middle. This is useful to write large numbers in a more readable way: `100000` can be written as `100_000`.
+
+```{.crystal, .crystal-play}
+p! 1, typeof(1)
+p! 1.0, typeof(1.0)
+p! 100_000, typeof(100_000)
+p! 100_000.0, typeof(100_000.0)
+```
+
+Float values print with with a decimal point. Integer values don't.
+
+!!! info
+    There are quite a few more numeric types, but most of them are intended only for special use cases such as binary protocols,
+    specific numeric algorithms and performance optimization. You probably don't need them for every-day programs.
+
+    See [Integer literal reference](../../syntax_and_semantics/literals/integers.md) and [Float literal reference](../../syntax_and_semantics/literals/floats.md)
+    for a full reference on all primitive number types and alternative representations.
+
+## Arithmetic
+
+### Equality and Comparison
+
+Numbers of the same numerical value are considered equal regarding the equality operator `==`, independent of their type.
+
+```{.crystal, .crystal-play}
+p! 1 == 1,
+   1 == 2,
+   1.0 == 1,
+   -2000.0 == -2000
+```
+
+Besides the equality operator, there are also comparison operators. They determine the relationship between two values.
+As with equality, comparability is also independent of type.
+
+```{.crystal, .crystal-play}
+p! 2 > 1,
+   1 >= 1,
+   1 < 2,
+   1 <= 2
+```
+
+The main comparison operator is `<=>`, also called *Spaceship operator* for its appearance. It compares its operands and returns a value that is either zero (both operands are equal),
+a negative value (the first operand is bigger), or a positive value (the second operand is bigger).
+
+```{.crystal, .crystal-play}
+p! 1 <=> 1,
+   2 <=> 1,
+   1 <=> 2
+```
+
+### Operators
+
+Basic arithmetic operations can be performed with operators. Most operators are *binary* (i.e. two operands), and
+written in infix notation (i.e. between the operands). Some operators are *unary* (i.e. one operand), and written in prefix
+notation (i.e. before the operand).
+The value of the expression is the result of the operation.
+
+```{.crystal, .crystal-play}
+p! 1 + 1,  # addition
+   1 - 1,  # subtraction
+   2 * 3,  # multiplication
+   2 ** 4, # exponentiation
+   2 / 3,  # division
+   2 // 3, # floor division
+   3 % 2,  # modulus
+   -1,     # negation (unary)
+   !4      # faculty (unary)
```

Comment:
Not sure what this is...  `!4` == "not 4" == `false`.  I think this might have been going for factorial, but I don't think there's an operator for that?